spec/Section 4 -- Introspection.md

```diff
@@ -222,6 +217,26 @@ enum __DirectiveLocation {
 }
 ```
 
+### The __Schema Type
+
+The `__Schema` type is returned from the `__schema` meta-field and provides
+all information about the schema of a GraphQL service.
+
+Fields\:
+
+* `description` may return a String or {null}.
+* `queryType` is the root type of a query operation, if supported.
```

Comment:
> if supported. Otherwise {null}.

WOW! It seemed to me that discussions of the optional `queryType` were about the fact that it is required for introspection, so it should always be? Did I miss something?

Reply:
Yes indeed, that one seems new :) @leebyron did you intend to make it nullable or is it a copy pasta? Until now a query type was mandatory.

Reply:
Definitely a mistake, thank you for the careful read!